pkg/autoscaler/autoscaler.go

```diff
@@ -538,7 +623,10 @@ var _ = Describe("Autoscaler should", framework.LabelAutoscaler, Serial, func()
 			uniqueJobName := fmt.Sprintf("%s-balance-nodegroups", workloadJobName)
 			By(fmt.Sprintf("Creating scale-out workload %s: jobs: %v, memory: %s",
 				uniqueJobName, jobReplicas, workloadMemRequest.String()))
-			workload := framework.NewWorkLoad(jobReplicas, workloadMemRequest, uniqueJobName, autoscalingTestLabel, targetedNodeLabel, "")
+			workload := framework.NewWorkLoad(jobReplicas, workloadMemRequest, uniqueJobName, autoscalingTestLabel, "", corev1.NodeSelectorRequirement{
+				Key:      targetedNodeLabel,
+				Operator: corev1.NodeSelectorOpExists,
+			})
```

Comment:
see row 702 (i cannot comment there): that test expects 6 nodes to be available on a cluster. Normally, this is true, but for multi-arch workflows we can have 7 or 8 (1 or 2 of a secondary arch)

Reply:
i'm a little confused by this comment. i get that we need to increase the maximum number, but is there something else i should be looking at?

Reply:
ignore it.... that's out of scope here and i should have deleted it as it was part of an offline conversation.